sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client.py

```diff
@@ -0,0 +1,248 @@
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+import os
+from azure.communication.administration import (
+    PhoneNumberAdministrationClient,
+    PstnConfiguration,
+    NumberUpdateCapabilities,
+    CreateSearchOptions
+)
+from phone_number_helper import PhoneNumberUriReplacer
+from _shared.testcase import (
+    CommunicationTestCase,
+    BodyReplacerProcessor
+)
+
+class PhoneNumberAdministrationClientTest(CommunicationTestCase):
+
+    def setUp(self):
+        super(PhoneNumberAdministrationClientTest, self).setUp()
+        self.recording_processors.extend([
+            BodyReplacerProcessor(
+                keys=["id", "token", "capabilitiesUpdateId", "phoneNumber", "phonePlanIds",
+                      "phoneNumberCapabilitiesUpdates",	"releaseId",
+                      "phoneNumberReleaseStatusDetails", "searchId", "phoneNumbers",
+                      "entities", "phonePlanGroups", "phonePlans", "phoneNumberCapabilitiesUpdate"]
+            ),
+            PhoneNumberUriReplacer()])
+        self._phone_number_administration_client = PhoneNumberAdministrationClient.from_connection_string(
+            self.connection_str)
+        if self.is_live:
+            self.country_code = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_COUNTRY_CODE')
+            self.locale = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_LOCALE')
+            self.phone_plan_group_id = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONE_PLAN_GROUP_ID')
+            self.phone_plan_id = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONE_PLAN_ID')
+            self.phone_plan_id_area_codes = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONE_PLAN_ID_AREA_CODES')
+            self.area_code_for_search = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_AREA_CODE_FOR_SEARCH')
+            self.search_id = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_SEARCH_ID')
+            self.search_id_to_purchase = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_SEARCH_ID_TO_PURCHASE')
+            self.search_id_to_cancel = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_SEARCH_ID_TO_CANCEL')
+            self.phonenumber_to_configure = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONENUMBER_TO_CONFIGURE')
+            self.phonenumber_to_get_config = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONENUMBER_TO_GET_CONFIG')
+            self.phonenumber_to_unconfigure = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONENUMBER_TO_UNCONFIGURE')
+            self.phonenumber_for_capabilities = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_PHONENUMBER_FOR_CAPABILITIES')
+            self.capabilities_id = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_CAPABILITIES_ID')
+            self.release_id = os.getenv('AZURE_COMMUNICATION_SERVICE_TNM_RELEASE_ID')
+            self.scrubber.register_name_pair(
+                self.phone_plan_group_id,
+                "phone_plan_group_id"
```

Comment:
This does not match the one you specified in playback mode
self.phone_plan_group_id = "phone-plan-group-id-1"
Please make sure it works.

Reply:
matched the srubber string with the playback mode string. thanks